SK.Domain/Enums/Kink.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace SK.Domain.Enums
+{
+    public enum Kink
```

Comment:
Enum is ok as long as this wont change. If enum will be saved in db then whenever you change order in enum you will have diferent values. 

Use enum static pattern or use kink as Defined ValueObject like this:
```
public abstract class Kink {
    public string Name { get; }

    protected Kink( string name){
    Name = name;
}
}

public class Ageplay : Kink {
    public Ageplay(string name) : base(name){
}
}
```

Or without abstract class just Kink. Without domain knowledge it's hard to advice.

Reply:
Topic of kinks will be discussed if there are any changes planned.